### PR TITLE
Update rooms-prep.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6302
Added info for PAC

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-prep.md [hunk 1]
```diff
@@ -92,6 +92,16 @@ Microsoft Teams Rooms is designed to inherit Proxy settings from the Windows OS.
     `"ProxyServer"="xx.xx.xx.xx:8080"`
     
     If ProxyServer doesn't exist you may have to add this key as a string, change the xx.xx.xx.xx:8080 to the ip/host and port of your Proxy server.
+ 
+    If customer is using a PAC file the configuration would look like the example below:
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-prep.md:96]:
```suggestion
    If the customer is using a PAC file the configuration would look like the example below:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
@flinchbot - are you able to tech review this change?

**Pull request merged**